### PR TITLE
Pulsar proxy configuration docs

### PR DESCRIPTION
**Pull request by lucperkins**
There's now a `proxy.conf` file that can be used to configure the Pulsar proxy. This PR adds those parameters to the docs.

### CODE REVIEW

**Comment by lucperkins** [comment]
@merlimat @aahmed-se This is ready for review

File: site/_data/config/proxy.yaml [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  description: Max concurrent outbound connections. The proxy will error out requests beyond that.
+- name: tlsEnabledInProxy
+  default: "false"
+  description: Whether TLS is enabled for the proxy
+- name: tlsEnabledWithBroker
+  default: "false"
+  description: Whether TLS is enabled when communicating with Pulsar brokers
+- name: tlsCertificateFilePath
+  description: Path for the TLS certificate file
+- name: tlsKeyFilePath
```
Comment (jai1) [inline site/_data/config/proxy.yaml]: - name: tlsTrustCertsFilePath
   description: Path for the trusted TLS certificate pem file.

File: site/_data/config/proxy.yaml [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  description: Path for the TLS private key file
+- name: tlsHostnameVerificationEnabled
+  default: "false"
+  description: Whether the hostname is validated when the proxy creates a TLS connection with brokers
+- name: tlsRequireTrustedClientCertOnConnect
+  default: "false"
+  description: Whether client certificates are required for TLS. Connections are rejected if the client certificate isn't trusted.
```
Comment (jai1) [inline site/_data/config/proxy.yaml]: Need a new line at the end.

File: site/_data/config/proxy.yaml (hunk 1 above)
Comment (jai1) [reply site/_data/config/proxy.yaml]: Add this as well ^^^^

**Comment by lucperkins** [comment]
@jai1 Done

**Comment by merlimat** [comment]
retest this please

**Comment by srkukarni** [comment]
retest this please

**Comment by srkukarni** [comment]
retest this please

**Pull request merged**